src/ivoc/ocpicker.cpp

```diff
@@ -109,13 +104,12 @@ void StandardPicker::event(const Event& e) {
 }
 
 void StandardPicker::unbind(int m, EventButton eb) {
-    long cnt = handlers_[m]->count();
-    long i, j;
-    for (i = 0, j = 0; i < cnt; ++i) {
-        ButtonHandler& b = *handlers_[m]->item(j);
-        if (b.eb_ == Event::any || b.eb_ == eb) {
-            delete handlers_[m]->item(j);
-            handlers_[m]->remove(j);
+    long cnt = handlers_[m]->size();
+    for (long i = 0, j = 0; i < cnt; ++i) {
+        ButtonHandler* b = handlers_[m]->at(j);
+        if (b->eb_ == Event::any || b->eb_ == eb) {
+            delete b;
+            handlers_[m]->erase(handlers_[m]->begin() + j);
```

Comment:
I'm sure we can do something less errorish

Reply:
Replying with what you figured out yourself. In a separate PR you can replace this with: a for-loop that deletes the elements and an `std::remove_if` all the resulting `nullptr`s. Maybe it can even be golfed a bit more.

Reply:
Coming in next PR